api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,21 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+    
+def query_one_bouquets(bouquet_id):
+    one_bouquet = BouquetsModel.query.filter_by(id=bouquet_id).first()
+    get_bouquet = False
+    if one_bouquet:
+        get_bouquet = BouquetsSchema(many=False).dump(one_bouquet)
+
+    return get_bouquet
+
+def is_integer(bouquet_id):
```

Comment:
I think this function is rather overkill. Instead use the builtin methods to check if value is int or not.